File: Arnav Sharma/Assignment-0/if-else/grade.java
// grading system
import java.util.Scanner;

public class grade {

        public static void main(String[] args){
            Scanner s = new Scanner(System.in);
            System.out.println("Enter your marks");
            int x = s.nextInt();
            if(x<25){
                System.out.println("F");
            }
            else if((x>=25)&&(x<45)){
                System.out.println("E");
            }
            else if((x>=45)&&(x<50)){
                System.out.println("D");
            }
            else if((x>=50)&&(x<60)){
                System.out.println("C");
            }
            else if((x>=60)&&(x<80)){
                System.out.println("B");
            }
            else if((x>=80)&&(x<=100)){
                System.out.println("A");
            }
            else{
                System.out.println("Not correct marks");
            }
            s.close();
        }
    }

File: Jay Parikh/if else/Grade.cpp
#include <iostream>
using namespace std;

int main(){
    int marks;
    cout<<"Enter the your marks \n";
    cin>>marks;
    if(marks<25){
        cout<<"F";
    }
    else if(marks<45 && marks>=25){
        cout<<"E";
    }
    else if(marks<50 && marks>=45){
        cout<<"D";
    }
    else if(marks<60 && marks>=50){
        cout<<"C";
    }
    else if(marks<80 && marks>=60){
        cout<<"B";
    }
    else if(marks>80){
        cout<<"A";
    }
    return 0;
}